gtsam/geometry/tests/testPose3.cpp

```diff
@@ -900,17 +900,6 @@ TEST(Pose3, ExpmapDerivative4) {
   }
 }
 
-TEST( Pose3, ExpmapDerivativeQr) {
```

Comment:
I should actually look into detail why this test failed. It might be invoking nearZero for not small enough values.